README.md

```diff
@@ -9,13 +9,31 @@
 [![IRC Channel](https://img.shields.io/badge/irc-%23calico-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#calico)
 <!--- end of master only -->
 
-# Calico on Docker
-As well as providing networking for OpenStack VMs, Calico can provide
-networking for containers in a Docker environment.  Each container gets its 
-own IP and fine grain security policy.  In addition, Calico can be deployed 
-without encapsulation or overlays to provide high performance at massive 
-scales.  For more information on Project Calico see 
-http://www.projectcalico.org/learn/.
+# Calico on Containers
+Calico provides a highly scalable networking solution for connecting data 
+center workloads (containers, VMs, or bare metal).  It is based on the same 
+scalable IP networking principles as the internet: providing connectivity using
+a pure Layer 3 approach.  Calico can be deployed without encapsulation or 
+overlays to provide high performance at massive scales.
+
+Read more about it on the [Project Calico website](http://www.projectcalico.org).
+
+When using Calico networking for containerized environments, each container
+gets its own IP and fine grain security policy.  A calico-node service runs
+on each node which handles all of the necessary IP routing, installation of 
+policy rules, and distribution of routes across the cluster of nodes.
+
+The calico-docker repository contains:
+-  The components for the `calico-node` container, comprising
+  * Felix, the Calico worker process
+  * BIRD, the route distribution process

```

Comment:
The repo doesn't contain felix, bird or confd. Maybe if you said "The configuration for the components of the calico-node container"